src/schemathesis/runner/impl/solo.py

```diff
@@ -17,37 +18,41 @@ def _execute(self, results: TestResultSet) -> Generator[events.ExecutionEvent, N
         auth = get_requests_auth(self.auth, self.auth_type)
         with get_session(auth) as session:
             for endpoint, test in self.schema.get_all_tests(network_test, self.hypothesis_settings, self.seed):
+                for case_hook in get_add_case_hooks():
```

Comment:
If `get_add_case_hooks` is here, then it will double the number of hypothesis tests. I am not sure that it is what you described earlier. Each test will be on a separate CLI line with the same name. I thought that you wanted a single extra request that is executed after the original one. All these requests will be executed after N hypothesis tests related to the previous endpoint. What do you think? Is it the kind of behavior you wanted?

Reply:
Yes I agree. I overlooked that several cases may be generated in `run_test`. I moved the hook to `network_test` and `wsgi_test`.